packages/location_background/example/pubspec.yaml

```diff
@@ -0,0 +1,59 @@
+name: location_background_plugin_example
+description: Demonstrates how to use the location_background_plugin plugin.
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^0.1.0
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+  location_background_plugin:
```

Comment:
Should this be a regular dependency?

Reply:
Probably. Not sure how that ended up there.